project/src/components/places/places.tsx

```diff
@@ -0,0 +1,39 @@
+import Map from '../map/map';
+import Sorting from '../sorting/sorting';
+import { useState } from 'react';
+import {City, Point} from '../../types/map';
+import { Offers } from '../../types/offer';
+import OffersList from '../offers-list/offers-list';
+
+type PlacesProps = {
+  location: City,
+  offers: Offers,
+  placesCount: number
+}
+
+function Places({ location, offers, placesCount }: PlacesProps) {
+  const [selectedPoint, setSelectedPoint] = useState<Point | undefined>(
+    undefined,
+  );
```

Comment:
```suggestion
  const [selectedPoint, setSelectedPoint] = useState<Point | null>(
    null,
  );
```
undefined лучше не используй

Reply:
при смене на null сразу сыпятся ошибки

Reply:
исправила